client/sites-dashboard/controller.tsx

```diff
@@ -17,6 +18,7 @@ export function sitesDashboard( context: PageJSContext, next: () => void ) {
 	context.primary = (
 		<>
 			<Global styles={ globalStyles } />
+			<QuerySites allExcerpts />
```

Comment:
Should we also `<QuerySites allSites />` here, to make sure the `state.sites.items` cache is primed